src/language/en-US.json

```diff
@@ -772,10 +772,13 @@
   "login_account": "Login",
   "login": "Login",
   "create_a_new_account": "Create a new account",
-  "Access_My_Account": "Access My Account",
-  "Do_not_have_Account": "You don't have an account yet?",
```

Comment:
Why did you remove this? This sentence is necessary.

Reply:
https://github.com/nemfoundation/symbol-desktop-wallet/issues/29 
how to work ?

Reply:
Thank you for your correction，I will review my code more carefully。